modules/compose.mjs

```diff
@@ -477,7 +488,7 @@ class ReplyWithHeader {
             } else {
                 let rwhHeaders = '';
                 for (let [key, value] of Object.entries(remainingHeaders)) {
-                    rwhHeaders += '<p style="margin:0cm;font-size:11.5pt"><span><b>'
+                    rwhHeaders += '<p style="margin:0"><span><b>'
```

Comment:
@ygoe Please refer to the above comment.

Reply:
Same as above.